code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -544,6 +544,8 @@
 	products = list(
 		"Surplus Special Equipment" = list(
 			/obj/item/beacon/supply_beacon = 1,
+			/obj/item/storage/box/tx29_system = 1,
+			/obj/item/ammo_magazine/rifle/tx29 = 6,
```

Comment:
I reccomend instead of a req implementation to see how the placable MG does it, ie limited amount in pfc weapon vendors

Reply:
I had it in the pfc vendor before the last commit, Nova said David asked for it to be in the ops vendor instead though.

Reply:
Back in unga vendor. David never actually replied to me when I asked why he wanted it in the ops vendor.